schema/spacewalk/upgrade/susemanager-schema-5.1.1-to-susemanager-schema-5.1.2/001-add-slfo-package-keys.sql

```diff
@@ -0,0 +1 @@
+insert into rhnPackageKey (id, key_id, key_type_id, provider_id) (select sequence_nextval('rhn_pkey_id_seq'), 'fec28eaf09d9ea69', lookup_package_key_type('gpg'), lookup_package_provider('SUSE LLC') from dual where not exists (select 1 from rhnPackageKey where key_id = 'fec28eaf09d9ea69'));
```

Comment:
Note, this is probably going to be needed to support SLE Micro 6.1, so if we add support before merging this PR, this needs to be done on `master`, and this will need to be removed from this PR.